source/localizable/tutorial/acceptance-test.md

```diff
@@ -0,0 +1,145 @@
+Lets think through what we want to do on the home page of our Super Rentals application.
+
+We want our application to:
+
+* List available rentals.
+* Display some key information about each rental in the list.
+* Link to information about the company.
+* Link to contact information.
+* Filter the list of rentals by city.
+
+We can represent these goals as [ember acceptance tests](../testing/acceptance/).
+The purpose of ember acceptance tests are to provide UI automated tests from a user perspective.
+They interact with the rendered webpage as an actual user of the system would.
+Acceptance tests are well suited for testing high level goals of your application, such as the ones listed above for Super Rentals.
```

Comment:
For the entire paragraph above, maybe we can simplify it down as follow?

We can ensure that our app is working as expected using acceptance tests.  Acceptance tests interact with our app like an actual person would, but can be automated, ensuring that our app doesn't break in the future.